ztools/x509/testdata/name.constraint.test.cert

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Where did this certificate come from? IIRC, you can add random text data to the bottom of the cert without the parse functionality failing, it's probably worth describing what this certificate is, or at least appending the output `openssl x509 -text -noout` for this certificate.